fdb-record-layer-lucene/src/test/java/com/apple/foundationdb/record/lucene/FDBLuceneIndexFailureTest.java

```diff
@@ -192,10 +192,11 @@ void repartitionGroupedTestWithExceptionMapping(boolean useLegacyAsyncToSync, bo
         }
 
         // run partitioning without failure - make sure the index is still in good shape
+        timer.reset();
         explicitMergeIndex(index, contextProps, schemaSetup, false, 0);
         try (FDBRecordContext context = openContext(contextProps)) {
             schemaSetup.accept(context);
-            assertEquals(2, getCounter(context, LuceneEvents.Counts.LUCENE_REPARTITION_CALLS).getCount());
+            assertEquals(1, getCounter(context, LuceneEvents.Counts.LUCENE_REPARTITION_CALLS).getCount());
```

Comment:
@ohadzeliger, please verify this code (time reset, then getting only one repartition call. With this PR, the are more retries than the old test anticipated)